package.json

```diff
@@ -71,7 +72,8 @@
       "/ranges/",
       "/index.js",
       "/preload.js",
-      "/range.bnf"
+      "/range.bnf",
+      "/benchmarks"
```

Comment:
`npm run template-oss-apply` will apply this and rebuild the .gitignore.

Reply:
I ran but I removed the changes on `.github/workflows` about erasing a folder called `release`.

Reply:
your local version may have been out of date. looks like the linting is green now which means we should be good now.